<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameForm</class>
 <widget class="QWidget" name="GameForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="blackNameLabel">
       <property name="styleSheet">
        <string notr="true">font: italic 28pt &quot;Brush Script MT&quot;;</string>
       </property>
       <property name="text">
        <string>Black</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="gameInfoLabel">
       <property name="styleSheet">
        <string notr="true">font: italic 28pt &quot;Brush Script MT&quot;;
color:rgb(14, 159, 255)</string>
       </property>
       <property name="text">
        <string>White begins!</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="whiteNameLabel">
       <property name="styleSheet">
        <string notr="true">font: italic 28pt &quot;Brush Script MT&quot;;</string>
       </property>
       <property name="text">
        <string>White</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="3">
      <widget class="CheckersView" name="boardView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="resizeAnchor">
        <enum>QGraphicsView::NoAnchor</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0" alignment="Qt::AlignHCenter|Qt::AlignTop">
      <widget class="QPushButton" name="mainMenuButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: italic 20pt &quot;Brush Script MT&quot;;
background-color: rgba(255,255,255,0);</string>
       </property>
       <property name="text">
        <string>Main menu</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckersView</class>
   <extends>QGraphicsView</extends>
   <header>checkersview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
